<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sidebar</class>
 <widget class="QWidget" name="sidebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="sidebar_name_label">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trickcal Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="home_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>내실 요약 (준비중)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/home.png</normaloff>icon/sidebar/home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">main_menu_group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="hero_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>사도</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/hero.png</normaloff>icon/sidebar/hero.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">main_menu_group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="equip_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>랭크작</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/equip.png</normaloff>icon/sidebar/equip.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">main_menu_group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="equip_sub" native="true">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="equip_abstract_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>랭크작 현황 한 눈에 보기</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">equip_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="equip_1_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>착용 장비 및 목표 설정</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">equip_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="equip_2_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>보유 장비 재료</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">equip_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="equip_3_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>파밍 동선 계산</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">equip_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="crayon_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>보드</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/crayon.png</normaloff>icon/sidebar/crayon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">main_menu_group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="crayon_sub" native="true">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="crayon_abstract_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>전체 보드 현황</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">crayon_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="crayon_1_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>보드작 현황 기록</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">crayon_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="crayon_2_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>최소 비용 보드판</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">crayon_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="dps_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DPS 계산기</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/dps.png</normaloff>icon/sidebar/dps.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">main_menu_group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cash_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>과금 효율 계산기</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/cash.png</normaloff>icon/sidebar/cash.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">main_menu_group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="cash_sub" native="true">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="cash_1_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>기본 설정</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">cash_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cash_2_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>계산하기</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">cash_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cash_3_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>이 버튼 삭제할까?</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">cash_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>177</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="account_group">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>계정</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="account_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="account_setting_btn">
        <property name="font">
         <font>
          <family>ONE Mobile POP</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>계정 관리</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setting_btn">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>설정</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/sidebar/setting.png</normaloff>icon/sidebar/setting.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="equip_group"/>
  <buttongroup name="cash_group"/>
  <buttongroup name="crayon_group"/>
  <buttongroup name="main_menu_group"/>
 </buttongroups>
</ui>
